### ISSUE TITLE
update list of waypoints after scanning for a formula and generating a waypoint

### ISSUE THREAD

**Issue by tinkus-gc** [cgeo/cgeo#13751]
### Describe your problem!

After using the function "Scan for waypoint formulas" and selecting a formula a waypoint is generated. But this waypoint won't appear on the list of waypoints.

Workaround: Reload the cache or update the cache listing.

### How to reproduce?

- open the cache listing of GC9NWMK
- call "Scan for waypoint formulas"
- select the found pattern and confirm with OK
- go to the waypoints tab

### Actual result after these steps?

the generated waypoint is not shown in the list

### Expected result after these steps?

the generated waypoint should show up in the list of waypoints

### Reproducible

Yes

### c:geo Version

2022.11.25

### System information

```text
Device:
-------
- Device type: Pixel 4a (5G) (bramble, google)
- Available processors: 8
- Android version: 13
- Android build: TQ1A.221205.011
- Screen resolution: 1080x2078px (411x792dp)
- Pixel density: 2.625
- System font scale: 1.0 / used scale: 1.0
- Sailfish OS detected: false
- Google Play services: enabled - 22.46.17 (190400-491726958)
```

**Comment by Lineflyer** [comment]
Had no time to check yet, but does this really need a refresh of the cache or is it enough to swipe to another tab and back to waypoints?

**Comment by bekuno** [comment]
I confirm the issue.
If the cache is saved, then it is enought to close and reopen the listing.

**Comment by eddiemuc** [comment]
Fixed with PR #13912
